user-guide/Cloud_Platform/CloudAdminApp/Granting_admin_consent.md

```diff
@@ -45,5 +45,6 @@ To grant admin consent and configure your tenant:
 > [!NOTE]
 >
 > - You can **revoke the permissions given to Skyline Communications at any time in the Azure Portal**. For more information, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/manage-application-permissions?pivots=portal).
-> - You can unlink the tenant by clicking the x next to the tenant ID. However, **unlinking the tenant** from the organization **will not remove the permissions given to Skyline**.
-> - If there are **changes to the software** that cause new or different permissions to be required, you will need to **repeat** the grand admin consent procedure in order to also provide consent for those permissions. You can do this by **unlinking** your tenant first.
+> - You can unlink the tenant by clicking the *Remove* button next to the tenant ID. However, **unlinking the tenant** from the organization **will not remove the permissions given to Skyline Communications**.
+> - If there are **changes to the software** that cause new or different permissions to be required, you can grant the permissions again by clicking the *Regrant* button next to the tenant ID.
```

Comment:
```suggestion
> - If there are **changes to the software** that cause new or different permissions to be required, you can grant admin consent again by clicking the *Regrant* button next to the tenant ID.
```

think this is more clear (because it's not giving permissions again, it's adding new permissions mainly)